core/SConscript.firmware

```diff
@@ -872,6 +884,16 @@ elif 'STM32U5G9xx' in CPPDEFINES_HAL or 'STM32U585xx' in CPPDEFINES_HAL:
 else:
     raise Exception("Unknown MCU")
 
+if STORAGE_INSECURE_TESTING_MODE:
+    INSECURE_TESTING_MODE_STR = """
+#########################################################
+#                                                       #
+#   STORAGE_INSECURE_TESTING_MODE enabled, DO NOT USE   #
+#                                                       #
+#########################################################
```

Comment:
Looks like this on my system:
```
#########################################################
# #
# STORAGE_INSECURE_TESTING_MODE enabled, DO NOT USE #
# #
#########################################################
```
Maybe just use more `#` instead of spaces?

Reply:
Shouldn't the next line be something like `action_bin.append('$ECHO $INSECURE_TESTING_MODE_STR')`?

Reply:
> Shouldn't the next line be something like `action_bin.append('$ECHO $INSECURE_TESTING_MODE_STR')`?

you would think so but apparently not :) then you'd get the text in the output twice, misformatted exactly the same way, once prepended with "echo"

(`#` is a shell comment so the lines above do execute cleanly in shell. i tried a couple things with `echo` and quoting, for whatever reason I didn't stumble on the correct combination and didn't really want to spend more time on it, and i'll just remove the misformatted lines)

Reply:
1728f6411f85